ddl/ddl_worker.go

```diff
@@ -950,6 +950,18 @@ func (w *worker) countForError(err error, job *model.Job) error {
 	return err
 }
 
+func (w *worker) processJobPausingRequest(d *ddlCtx, job *model.Job) (err error, isRunnable bool) {
+	if job.IsPaused() {
+		logutil.Logger(w.logCtx).Debug("[ddl] paused DDL job ", zap.String("job", job.String()))
```

Comment:
Why change the log  level to debug?

Reply:
It would be much since every time of polling would come to here if paused.